scripts/common/containerd.sh

```diff
@@ -14,6 +14,12 @@ function containerd_patch_for_minor_version() {
     echo ""
 }
 
+# containerd_node_is_using_docker returns 0 if the current node is using docker as the container runtime.
+function containerd_node_is_using_docker() {
+    local node=$(hostname | tr '[:upper:]' '[:lower:]')
```

Comment:
is it possible to use the utility function get_local_node_name?

Reply:
Sure, done.